### ISSUE TITLE
Upgrade to Django 1.8

### ISSUE THREAD

**Issue by harrislapiroff** [littleweaver/django-brambling#443]
It has cool stuff!

**Comment by melinath** [comment]
This is blocked by alex/django-filter#227.

**Comment by melinath** [comment]
Django-filter has FINALLY released a new version, so we could upgrade to 1.8 (potentially even skip straight to 1.9?)

**Comment by harrislapiroff** [comment]
woot woot :tada:

**Comment by emilyhorsman** [comment]
😂

**Comment by emilyhorsman** [comment]
Closing in favour of #861
